src/analysis/retail/priest/discipline/modules/guide/RaptureAnalysis.tsx

```diff
@@ -0,0 +1,374 @@
+              </span>
+            </div>
+          </div>
+        </>
+      );
+
+      const badDamageCastTooltip = (index: number) => (
+        <>
+          Once you have applied <SpellLink id={TALENTS_PRIEST.ATONEMENT_TALENT.id} />
+          s, it is important to get good value out of them by using many high damage spells. Casting
+          non damage spells should be avoided where possible.
+        </>
+      );
+
+      const badTeTooltip = () => (
+        <>
+          Cast not buffed by opposite spell school. Make sure to plan your damage rotation with{' '}
+          <SpellLink id={TALENTS_PRIEST.TWILIGHT_EQUILIBRIUM_TALENT.id} />
+        </>
+      );
+
+      const goodTeCast = (index: number) => {
+        // first cast cannot be buffed by TE - The buff is too short.
+        if (index === 0) {
+          return true;
+        }
+
+        if (TE_GUIDE_FILTER.includes(ramp.damageRotation[index].ability.guid)) {
+          //first cast is holy
+          if (isHolySpell(ramp.damageRotation[index].ability.guid)) {
+            // second cast also holy
+            if (isHolySpell(ramp.damageRotation[index - 1].ability.guid)) {
+              return false;
+            }
+          }
+
+          // first cast is shadow
+          if (isShadowSpell(ramp.damageRotation[index].ability.guid)) {
+            // second cast also Shadow
+            if (isShadowSpell(ramp.damageRotation[index - 1].ability.guid)) {
+              return false;
+            }
+          }
+        }
+
+        return true;
+      };
+
+      const teTooltip = (index: number) => {
+        const spell = ramp.damageRotation[index].ability.guid;
+        return TE_SPELLS().includes(spell) && goodTeCast(index) ? '' : badTeTooltip();
+      };
+
+      const damageSequence = ramp.damageRotation.map((cast, index) => {
+        const tooltipContent = () => {
+          const badCast = ATONEMENT_DAMAGE_SOURCES[cast.ability.guid]
+            ? null
+            : badDamageCastTooltip(index);
+          const badTe = TE_SPELLS().includes(cast.ability.guid) ? teTooltip(index) : null;
+
+          if (!badCast && !badTe) {
+            return <>No issues found.</>;
+          } else {
+            return (
+              <>
+                {badCast}
+                {badTe}
+              </>
+            );
+          }
+        };
+
+        const iconClass = `evang__icon ${
+          ATONEMENT_DAMAGE_SOURCES[cast.ability.guid] ? '' : '--fail'
+        } ${TE_SPELLS().includes(cast.ability.guid) && goodTeCast(index) ? '' : '--warning'}`;
+
+        return (
+          <Tooltip content={tooltipContent()} key={index} direction="up">
+            <div className="" data-place="top">
+              <Icon icon={cast.ability.abilityIcon} className={iconClass} />
+            </div>
+          </Tooltip>
+        );
+      });
+
+      return (
+        <ControlledExpandable
+          header={header}
+          element="section"
+          expanded={!isExpanded}
```

Comment:
change the default value for `isExpanded` rather than inverting the logic here
```suggestion
          expanded={isExpanded}
```